Components/Core/Inc/Timer.hpp

```diff
@@ -15,6 +15,13 @@ constexpr uint32_t DEFAULT_TIMER_COMMAND_WAIT_PERIOD = MS_TO_TICKS(15); // Defau
 
 #define DEFAULT_TIMER_PERIOD (MS_TO_TICKS(1000)) // 1s
 
+enum CurrentState {
```

Comment:
Please rename enum to TimerState or TimerStatus, this hpp will be in many files with state.

Reply:
Done